browser/components/MarkdownPreview.js

```diff
@@ -1036,9 +1026,10 @@ export default class MarkdownPreview extends React.Component {
 
     if (!href) return
 
-    const regexNoteInternalLink = /main.html#(.+)/
+    const extractId = /(main.html)*#/
```

Comment:
Shouldn't it be `/(main.html)?#/` ?

Reply:
No, I think I need this line.
First, the `id` of the target element don't have a hash.
e.g.
![FixedSlug](https://user-images.githubusercontent.com/11808736/57393456-18031d80-71fe-11e9-93b2-3c8bec2c07f8.jpg)
Therefore, I had to remove a hash in `linkHash`.

Second, the past `regexNoteInternalLink` didn't match `linkHash`.
`linkHash` doesn't include `main.html`.
e.g.
![FixRegExp](https://user-images.githubusercontent.com/11808736/57577793-cd152e80-74b9-11e9-8919-35facbed2a84.jpg)
So, I had to change the regex rule.

Reply:
I understand that, it's just I want to replace the regex from `(main.html)*#` to `(main.html)?#`. But I think it doesn't matter :smile: We can go by your way then.

Reply:
I'm sorry. I misunderstood that this line was not necessary.
Thank you for a suggestion. I think `(main.html)?#` is better than `(main.html)*#` too.